scripts/generateSolanaApiTypes.js

```diff
@@ -0,0 +1,149 @@
+/* eslint-disable no-console */
+// npx openapi-typescript https://deep-index.moralis.io/api-docs/v2/swagger.json --output types/generated/SolanaApi.d.ts
+
+// const swaggerJsdoc = require('swagger-jsdoc');
+const path = require('path');
+const fs = require('fs');
+const https = require('https');
+const openapiTS = require('openapi-typescript').default;
+
+// URL will be changed when api is deployed
+const API_HOST = '';
+const SWAGGER_PATH = '/api-json';
```

Comment:
```suggestion
const API_HOST = 'solana-gateway.moralis.io';
const SWAGGER_PATH = '/api-json';
```
The API_HOST needs to be set